### PR TITLE
Cut v2.27.0-rc.0

### PR DESCRIPTION
**Pull request by csmarchbanks**
This is for review to make sure I did not miss anything. Before I merge this PR and actually do the release I will merge https://github.com/prometheus/prometheus/pull/8792 into main and update release-2.27 accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+* [FEATURE] Promtool: Retroactive rule evaluation functionality. #7675
+* [FEATURE] Configuration: Environment variable expansion for external labels. Behind `--enable-feature=expand-external-labels` flag. #8649
+* [FEATURE] TSDB: Add a flag(`-storage.tsdb.max-chunk-size`) to control chunk allocation size for small Prometheus instances. #8478
+* [FEATURE] UI: Add a dark theme. #8604
+* [FEATURE] AWS Lightsail Discovery: Add AWS Lightsail Discovery. #8693
+* [FEATURE] OAuth: Allow OAuth 2.0 to be used anywhere an HTTP client is used. #8761
+* [FEATURE] Remote Write: Send exemplars via remote write. Experimental and disabled by default. #8296
+* [ENHANCEMENT] Digital Ocean Discovery: Add `__meta_digitalocean_vpc` label. #8642
+* [ENHANCEMENT] Scaleway Discovery: Read Scaleway secret from a file. #8643
+* [ENHANCEMENT] Docker Discovery: Add Docker Service Discovery. #8629
```
Comment (roidelapluie) [inline CHANGELOG.md:12]: This is a feature.

Comment (csmarchbanks) [reply CHANGELOG.md]: Thanks, fixed!

**Review by bwplotka — APPROVED** [review]
Amazing, let's go!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
+## 2.27.0-rc.0 / 2021-05-09
+
+* [FEATURE] Promtool: Retroactive rule evaluation functionality. #7675
+* [FEATURE] Configuration: Environment variable expansion for external labels. Behind `--enable-feature=expand-external-labels` flag. #8649
+* [FEATURE] TSDB: Add a flag(`-storage.tsdb.max-chunk-size`) to control chunk allocation size for small Prometheus instances. #8478
```
[the remark this answers is not in the document]
Comment (codesome) [reply CHANGELOG.md:5]: (I have thus updated the PR description)

[the remark this answers is not in the document]
Comment (roidelapluie) [reply CHANGELOG.md:5]: Yes, just update that in the same PR as full release

**Comment by roidelapluie** [comment]
prombench LGTM

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (csmarchbanks) [reply CHANGELOG.md:5]: I have updated the description in https://github.com/prometheus/prometheus/pull/8814, please let me know if it does not look correct.

**Pull request merged**